### ISSUE TITLE
Global Page Translation templates

### ISSUE THREAD

**Issue by svenseeberg** [digitalfabrik/integreat-cms#2081]
### Motivation
For region managers it is often not easy to onboard additional editors. Therefore, text is often created in word documents instead of directly in the system. This is also due to lack of trust.

### Proposed Solution
We need to make it easier for external contributors to create texts that fulfill basic requirements and can directly be used for Integreat. We can create (a simple) page template system that helps creating a text that is suitable for Integreat.

- The network management needs a management for the templates. Templates consist of a name and a field for the HTML source code. We do not need a WYSIWYG editor here, as we can simply copy the source code from other pages.
- The editor  needs a new menu to choose from available templates and insert them into the current page. The text should be inserted at the current cursor position (and not overwrite existing page content). But usually it should be used in empty pages.

### Alternatives
N/A

### Additional Context
Together with #316 editors can easily create a simple draft page and create an external link for external contributors to improve the text.

### Design Requirements
N/A

**Comment by svenseeberg** [comment]
There is one open question: do we need a reference to a language in the templates? Maybe we don't. The title could be sufficient.

**Comment by osmers** [comment]
@svenseeberg yes, I think it is not really a relevant scenario - maybe it would be prudent to only have templates in the German page editor?

**Comment by svenseeberg** [comment]
I think we should keep it easy and ignore languages. We can introduce them later, if we have to.

**Comment by osmers** [comment]
I agree - also, it wouldn't be too difficult to just get them translated, right?

**Comment by svenseeberg** [comment]
Yes, we could do this with automatic translations. However, I would still initially keep the functionality as close to the bare minimum as we can. That means ignoring languages at the start and test if the feature is useful at all.

**Comment by osmers** [comment]
Sounds good to me :) now we just need to develop a template or two, right?

**Comment by dkehne** [comment]
i do not see that this is the solution for the mentioned problem. I would not develop such templates but start with https://github.com/digitalfabrik/integreat-cms/issues/316 and see how this improves the process already.

people who feel more comfortable to create some content on their local machine first, will always do that no matter how easy the cloud-solution is. this can only be changed by training and showing the benefits, not by using a more UI which is more simple.

**Comment by svenseeberg** [comment]
@osmers can you discuss Daniel's comment within the service team?

**Comment by PeterNerlich** [comment]
closed as idea
